### ISSUE TITLE
Remove etcd module once it is merged to ansible

### ISSUE THREAD

**Issue by LaurentValette** [cloudtrust/runtime#4]
Commit 598baaf added the ansible etcd module, retrieved from https://github.com/jparrill/ansible-module-etcd

This module is in the process of being merged into the official ansible modules: See issue ansible/ansible#25546. After the merge, we should use this version of the module and remove it from library/etcd.py.
